wikimedia/wikipedia-ios
# Add Xcode Cloud tag script

**Phabricator:** 
N/A

### Notes
This PR adds a post-build script to tag the branch with "betas/{nnnn}". As of this time there is no corresponding GitHub action, it uses the build number that Xcode Cloud determines and increments. Xcode Cloud is already configured to kick off a nightly build against `main`, this PR is just for adding the tag if it successfully archives.

This PR also includes a Ruby/Gems update, so that I could commit the new build script (we have a git commit hook written in Ruby). 

### Test Steps
1. Go to the Nightly Build workflow in the Xcode Cloud area within Xcode.
2. Right-click Nightly Build and choose "start workflow" to kick off manually. Choose this PR branch (`xcode-cloud-tag-script`).
3. Confirm the build successfully deploys to App Store Connect, and the "betas/nnnn" tag is successfully added to the latest commit in this branch.
